trio/_util.py

```diff
@@ -224,10 +240,10 @@ def decorator(func: CallT) -> CallT:
     return decorator
 
 
-def fixup_module_metadata(module_name, namespace):
-    seen_ids = set()
+def fixup_module_metadata(module_name: str, namespace: dict[str, t.Any]) -> None:
+    seen_ids: set[int] = set()
 
-    def fix_one(qualname, name, obj):
+    def fix_one(qualname: str, name: str, obj: t.Any) -> None:
```

Comment:
The function body assumes nothing about the values in the dict, so might as well use `object` instead of `t.Any` to keep it that way.